libs/SoloPlay/Functions/MuleloggerOverrides.js

```diff
@@ -215,7 +215,7 @@ MuleLogger.logItem = function (unit, logIlvl, type = "Player") {
 		}
 
 		break;
-	case sdk.itemquality.Unique:
+	case item.unique:
```

Comment:
This case structure needs to be reverted back, it's checking the numerical value instead of the boolean one

Reply:
done